src/configureAWS.ts

```diff
@@ -43,7 +43,9 @@ function getCredentialsProviderChain(profile?: string) {
         });
     };
     return new aws.CredentialProviderChain(
-      [() => new aws.SharedIniFileCredentials({profile, tokenCodeFn, useCache: true})]);
+      [() => new aws.SharedIniFileCredentials({profile, tokenCodeFn, useCache: true}),
+       () => new aws.ProcessCredentials({profile})
```

Comment:
Double checking my assumption: Is this ordering intentional to mimic the setup that `aws-sdk-js` has for the DefaultProviders?

Reference [link](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/CredentialProviderChain.html#defaultProviders-property)

Reply:
Yep